app/components/learner-select-field.js

```diff
@@ -0,0 +1,6 @@
+import UISelectFieldComponent from 'ui-select/components/ui-select-field';
+import layout from '../templates/components/learner-select-field';
+
+export default UISelectFieldComponent.extend({
+  layout
```

Comment:
It's the ES6 syntax for object literal shorthand for property names that are same as value variable names.

```javascript
let name = 'Jon';
let hash = { name: name } ;
```
is equivalent to
```javascript
let hash = { name } ;
```